src/EnergyPlus/DataZoneEquipment.hh

```diff
@@ -143,6 +143,14 @@ namespace DataZoneEquipment {
 
 	// Types
 
+	enum class LoadDist 
+	{
+		sequentialLoading,
+		uniformLoading,
+		uniformPLRLoading,
+		sequentialUniformPLRLoading
```

Comment:
I know we don't have a consistent style guide yet, but I think in the future we will probably have capitalized enum constants.  No big deal for now since I don't think it's decided, just noting it.

Reply:
Changed to CapitalizeEnums